nnunet/utilities/file_endings.py

```diff
@@ -17,9 +17,11 @@
 
 
 def remove_trailing_slash(filename: str):
-    while filename.endswith('/'):
-        filename = filename[:-1]
-    return filename
+    return os.path.normpath(filename)
+
+
+def get_last_folder(filename: str):
```

Comment:
this should be foldername instead of filename and sh should check whether it actually is a folder and not a file

Reply:
I changed the variable name and added an assertion. I didn't verify that after this changes that everything is still working, since I assume you want to test it anyways to not accidentally destroy linux support ;-)

Reply:
@FabianIsensee I had to remove the assertion again, since the code was not working anymore (see discussion with user further down in the chat): could you crosscheck if there should really be just folders as input, particularly with the script `nnUNet_convert_decathlon_task` there seems to be filepathes passed through the function.